cmd/influxd/run/server_test.go

```diff
@@ -2330,7 +2323,7 @@ func TestServer_Query_LimitAndOffset(t *testing.T) {
 		&Query{
 			name:    "limit1 higher than MaxGroupBy but the number of data points is less than MaxGroupBy",
 			command: `select mean(foo)  from "limit"  where  time >= '2009-11-10T23:00:02Z' and time < '2009-11-10T23:00:03Z' group by time(1s), * fill(0)  limit 2147483647`,
-			exp:     `{"results":[{"series":[{"name":"limit","tags":{"tennant":"paul"},"columns":["time","mean"],"values":[["2009-11-10T23:00:02Z",2]]}]}]}`,
```

Comment:
I have a question about this test. I am not sure if the original test was actually wrong, and I think I have corrected it.